js/src/forum/index.js

```diff
@@ -47,10 +47,13 @@ app.initializers.add('flarum-markdown', function(app) {
         <MarkdownButton title={tooltip('header')} icon="fas fa-heading" style={{ prefix: '### ' }} />
         <MarkdownButton title={tooltip('bold')} icon="fas fa-bold" style={{ prefix: '**', suffix: '**', trimFirst: true }} hotkey="b" />
         <MarkdownButton title={tooltip('italic')} icon="fas fa-italic" style={{ prefix: '_', suffix: '_', trimFirst: true }} hotkey="i" />
+		<MarkdownButton title={tooltip('underline')} icon="fas fa-underline" style={{ prefix: '[u]', suffix: '[/u]', trimFirst: true }} hotkey="u" />
+	    <MarkdownButton title={tooltip('strikethrough')} icon="fas fa-strikethrough" style={{ prefix: '~~', suffix: '~~', trimFirst: true }} hotkey="s" />
         <MarkdownButton title={tooltip('quote')} icon="fas fa-quote-left" style={{ prefix: '> ', multiline: true, surroundWithNewlines: true }} />
         <MarkdownButton title={tooltip('code')} icon="fas fa-code" style={{ prefix: '`', suffix: '`', blockPrefix: '```', blockSuffix: '```' }} />
         <MarkdownButton title={tooltip('link')} icon="fas fa-link" style={{ prefix: '[', suffix: '](url)', replaceNext: 'url', scanFor: 'https?://' }} />
         <MarkdownButton title={tooltip('image')} icon="fas fa-image" style={{ prefix: '![', suffix: '](src)', replaceNext: 'src', scanFor: 'https?://' }} />
+        <MarkdownButton title={tooltip('center')} icon="fas fa-align-center" style={{ prefix: '[center]', suffix: '[/center]', trimFirst: true }} hotkey="c" />
```

Comment:
This extension only [configures the Litedown bundle](https://github.com/flarum/markdown/blob/20b6ce4ce8bb56a36c0abae16b30a2c020ee4a7a/extend.php#L20-L22), which is a simple Markdown flavor.

Therefore, we should not add buttons for BBCode-specific functionality here. If anything, that would belong in [our bbcode extension](https://github.com/flarum/bbcode) - however, that would first require [moving the button toolbar to core](https://github.com/flarum/core/issues/1872), so that it can be used from other extensions as well. That is something that will take us a while to figure out, so let's focus on the Markdown part for now.